src/stories/autosuggest-text/AutosuggestText.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { AutosuggestText } from "./AutosuggestText";
+
+export default {
+  title: "Components / Autosuggest - Text",
+  component: AutosuggestText,
+  decorators: [withDesign],
+  argTypes: {
+    items: {
+      name: "Items",
+      defaultValue: ["Item one", "Item two"],
+      control: { type: "array" }
+    }
+  },
+  parameters: {},
+} as ComponentMeta<typeof AutosuggestText>;
+
+const Template: ComponentStory<typeof AutosuggestText> = (args) => (
```

Comment:
To be consistent I think you sgould import the AutosuggestTextProps type and attach it to args: 
`(args: AutosuggestTextProps)`
Like it is done here:
https://github.com/danskernesdigitalebibliotek/dpl-design-system/blob/main/src/stories/breadcrumb/Breadcrumb.stories.tsx#L28

Reply:
done :)